test/basic.ml

```diff
@@ -1,20 +1,36 @@
 let digit = [%sedlex.regexp? '0' .. '9']
 let number = [%sedlex.regexp? Plus digit]
 
+let print_pos buf =
+  let f { Lexing.pos_lnum; pos_bol; pos_cnum; _ } =
+    Printf.sprintf "line=%d:bol=%d:cnum=%d" pos_lnum pos_bol pos_cnum
```

Comment:
a `line:column` format could be easier to read.
and matybe `line:column-line:column` like in error messages.

Reply:
I'm trying to report the actual internal parameters for easier debugging.